messaging/build.gradle

```diff
@@ -71,6 +71,7 @@ dependencies {
 
     implementation 'com.google.protobuf:protobuf-javalite:3.10.0'
 
+    implementation 'com.android.support:exifinterface:28.0.0'
```

Comment:
Drive-by, fixes a linter warning suggesting we use the support library version of ExifInterface